backend/app/graphql/user_mutations.py

```diff
@@ -1,27 +1,20 @@
+from app.resources.update_user_dto import UpdateUserDTO
 import graphene
 from .services import services
 from .types import (
     Mutation,
     MutationList,
+    User,
 )
-from app.resources.user_dto import UserDTO
-
-
-class User(graphene.ObjectType):
-    name = graphene.String()
-    email = graphene.String()
-    role = graphene.String()
 
 
+# TODO: refactor and update with user_service.update_user_by_id
```

Comment:
Is this TODO a question for me or a remainder for yourself?

Reply:
I'll interpret this as a question or a need for clarification. The purpose of this mutation is so that admin can change/update information about users, and users can update information about themselves. This information might be email, primary contact..etc. This means any of the information collected and stored in the UserInfo could change.

Hence, this mutation should have an extra argument other than auth_id and id that stands for the new updated UserInfo, which contains all the new information about the user, and any past information that we want to remain the same. 

This info should be passed to the update_user_by_id through the UpdateUserDTO.